async-nats/src/connector.rs

```diff
@@ -88,6 +89,16 @@ impl Connector {
         })
     }
 
+    pub(crate) fn reconnect_delay_callback_default(attempts: usize) -> Duration {
```

Comment:
Nit, make it a free standing function in this module instead of on Connector?

```suggestion
    pub(crate) fn reconnect_delay_callback_default(attempts: usize) -> Duration {
```